tasties_app/tests/test_ingredient.py

```diff
@@ -0,0 +1,68 @@
+import pytest as pytest
+
+from tasties_app.models import Ingredient, Recipe, Category
+from django.contrib.auth.models import User
+
+
+@pytest.fixture
+def ingredient():
+    """
+    Creates a user, a recipes, and an ingredient,
+    and adds them to the database.
+    returns: ingredient
+    """
+    user1 = User.objects.create_user('testuser', 'password')
+    category = Category.objects.create(category_name='Breakfast')
+    category.save()
+    recipe = Recipe(title='Test Recipe1', author_id=user1, description='Test Description1',
+                    directions='Test Directions1', minutes_to_make=1, recipe_picture='test_picture1')
+    recipe.save()
+    recipe.categories.add(category)
+    ingredient = Ingredient(recipe_id=recipe, amount=1.0, measurement_unit='WHOLE',
+                            description='test ingredient')
+    ingredient.save()
+    return ingredient
+
+
+class TestIngredientModel:
+    """
+    Tests the Ingredient model:
+        - Creation of an ingredient
+        - Editing of an ingredient
+        - Negative tests to ensure that invalid data cannot be added to the database
+    """
+    @pytest.mark.django_db
+    def test_create_data_fixture(self, ingredient):
```

Comment:
This test is checking that the ingredient is created, so maybe you can call it something like:
```suggestion
    def test_create_ingredient(self, ingredient):
```